src/garage/torch/algos/ddpg.py

```diff
@@ -116,12 +110,25 @@ def __init__(self,
 
         self.target_policy = copy.deepcopy(self.policy)
         self.target_qf = copy.deepcopy(self.qf)
-        self.policy_optimizer = optimizer(self.policy.parameters(),
-                                          lr=self.policy_lr)
-        self.qf_optimizer = optimizer(self.qf.parameters(), lr=self.qf_lr)
+        if policy_optimizer is None:
+            policy_optimizer = torch.optim.Adam(self.policy.parameters(),
```

Comment:
Can you add a launcher which actually uses a different optimizer? How do they know they should create the optimizer with self.policy.parameters()?

I think it might be better to pass the type in the constructor but have a better way of passing the optimizer args.

@krzentner WDYT?

Reply:
I added example of new API in ddpg launcher. 

People can know the `optimizer` type is `torch.optim.Optimizer` from algo documentation, and then they can refer to the torch documentation for further details.